exercises/sieve/sieve.py

```diff
@@ -1,2 +1,9 @@
 def sieve(limit):
-    pass
+    isprimes = [True] * (limit+1)
```

Comment:
I'm afraid that `sieve` already has example implementation - https://github.com/exercism/python/blob/master/exercises/sieve/example.py

And `sieve.py` is used a placeholder template which is deliver by CLI to the learners 

Feel free to ask any questions